docs/getting-started/1-concepts.md

```diff
@@ -4,18 +4,19 @@ The power of integrating linting into the design-first workflow, or any workflow
 
 To achieve this, Spectral has three key concepts:
 
-- **Rules** filter your object down to a set of target values and specify the function that is used to evaluate those values.
+- **Rules** filter your object down to a set of target values and specify the function (or functions) used to evaluate those values.
 - **Functions** accept a value and return issues if the value is incorrect.
 - **Rulesets** act as a container for rules and functions.
 
-Rules can be comprised of one of more functions, to standardize structured content, like making sure your OpenAPI descriptions match your [API style guides](https://stoplight.io/api-style-guides-guidelines-and-best-practices):
+Spectral comes bundled with a [set of core functions](../reference/functions.md) and rulesets for working with [OpenAPI v2 and v3](./4-openapi.md) and [AsyncAPI v2](./5-asyncapi.md) that you can chose to use or extend, but Spectral is about far more than just checking your OpenAPI/AsyncAPI documents are valid.
 
+By far the most popular use-case of Spectral is automating [API Style Guides](https://stoplight.io/api-style-guides-guidelines-and-best-practices?utm_source=github&utm_medium=spectral&utm_campaign=docs), implementing rules that your Architecture, DevOps, API Governance or "Center of Excellence" teams have decided upon. Companies generally write these style guides as wiki pages, and loads can be found on [API Style Book.com](http://apistylebook.com/), but most of these rules could be automated with Spectral.
```

Comment:
Can we add a link to the platform doc here?
https://meta.stoplight.io/docs/platform/71b92b2b5e388-style-guide-overview

Reply:
I think it might be too early on trying to explain what Spectral is, without confusing people with Stoplight Platform and various types of Style-Guide-The-Product stuff, but I'll try and work that link in a little later in the docs.